datalad/support/tests/test_github_.py

```diff
@@ -14,12 +14,31 @@
 import github as gh
 
 from ..exceptions import AccessDeniedError
-from ...tests.utils import assert_raises, assert_equal, eq_, assert_in
-
+from ...tests.utils import (
+    assert_equal,
+    assert_greater,
+    assert_in,
+    assert_raises,
+    eq_,
+    patch_config,
+    skip_if,
+    skip_if_no_network,
+)
+from ...consts import (
+    CONFIG_HUB_TOKEN_FIELD,
+)
 from ...utils import swallow_logs
 
 from .. import github_
-from ..github_ import get_repo_url
+from ..github_ import (
+    _gen_github_entity,
+    _get_github_cred,
+    get_repo_url,
+)
+
+
+skip_if_no_github_cred = \
+    lambda f: skip_if(cond=not _get_github_cred().is_known)(f)
```

Comment:
Is this lambda unnecessary?

Reply:
ha ha ha -- I was dancing around to make it work and have missed that it should be as simple as you suggested ;)  Thank you for the review -- force pushing!